Tools/autotest/arducopter.py

```diff
@@ -1488,7 +1488,7 @@ def fly_super_simple(self, timeout=45):
 
         # switch to stabilize mode
         self.change_mode("STABILIZE")
-        self.set_rc(3, 1500)
```

Comment:
This will make it climb very quickly!  even 1550 would keep it off the ground I think.